kmk/common/internal_keycodes.py

```diff
@@ -16,13 +16,54 @@ def process_internal_key_event(state, action, changed_key, logger=None):
         return tg(state, action, changed_key, logger=logger)
     elif changed_key.code == Keycodes.Layers._KC_TO:
         return to(state, action, changed_key, logger=logger)
+    elif changed_key.code == Keycodes.KMK.KC_GESC.code:
```

Comment:
We can get away with just checking `changed_key == KC_GESC` here, don't REALLY have to check the code (since `KC_GESC` is a constant, whereas things like `KC_MO` are generated objects and required that `_MO` code hackaround)

Reply:
I did in fact have to check .code here. When I did not, it did not match, and was actually bypassed. I manually triggered this function during testing until I sorted this out. Not sure why.